variables.tf

```diff
@@ -2,10 +2,40 @@
 # Input Variables
 ##############################################################################
 
-#variable "myvariable" {
-#  type        = string
-#  description = "A description of my variable"
-#  default     = ""
-#}
+variable "security_group_rules" {
+  description = "A list of security group rules to be added to the default vpc security group"
+  type = list(
+    object({
+      add_ibm_internal_sg_rules = optional(bool)
+      prepend_ibm_rules         = optional(bool)
```

Comment:
I don't think there is a need to give option to append or prepend rules for SG as there is no ordering in the rules (unlike the rules in network ACL)